SECURITY.md

```diff
@@ -5,8 +5,11 @@
 | Version | Supported          |
 | ------- | ------------------ |
 | < 2.2   | :x:                |
-| 2.2.x   | :white_check_mark: |
-| 3.0.x   | :white_check_mark: |
+| 2.2.x   | :x:                |
+| 3.0.x   | :x:                |
+| 3.1.x   | :white_check_mark: |
+| 4.0.x   | :white_check_mark: |
```

Comment:
I believe now that 4.1.x is released we won't support security fixes to 4.0.x, since I think our policy is to support the latest minor/patch version of the latest two major versions.

Reply:
+1